src/translators/sql_translator.py

```diff
@@ -73,7 +73,6 @@ def current_timex() -> str:
     return datetime.utcnow().isoformat(timespec='milliseconds')
 
 
-# TODO: use below getters everywhere rather than entity id and type strings!
```

Comment:
isn't this todo about using `entity_id` and `entity_type` instead of `dict.get('id')` and `dict.get('type')`?